test/commands/scanner/run.custom.test.ts

```diff
@@ -31,7 +31,8 @@ describe('scanner run with custom config E2E', () => {
 	});
 
 	it('Can use custom ESLint config to detect violations', () => {
-		const targetPath = path.join('.', 'test', 'code-fixtures', 'projects', 'ts', 'src', 'simpleYetWrong.ts');
+		const targetPath = '"' + path.join('.', 'test', 'code-fixtures', 'projects', 'ts', 'src', 'simpleYetWrong.ts') +
+			',' + path.join('.', 'test', 'code-fixtures', 'unparsable-json') + '"'; // Adding this to check that json files aren't picked up
```

Comment:
For TDD: This was the easy way to first add in a failing test. I added in an unparsable json file to the target which got picked up by the eslint-custom engine and caused a parse violation to show up (which is basically what the user is complaining about with #1146). Then I fixed this by making estling-custom have a configurable set of target files (like the other engines do) to only include what the other eslint* engines allow. This fixed then issue.